analyzer/src/main/kotlin/managers/Sbt.kt

```diff
@@ -213,3 +214,17 @@ class Sbt(
         // This is not implemented in favor over overriding [resolveDependencies].
         throw NotImplementedError()
 }
+
+private fun moveGeneratedPom(pomFile: File): File {
+    val targetDir = pomFile.absoluteFile.parentFile.searchUpwardsForSubdirectory("target")
+        ?: return pomFile
+    val targetFilename = pomFile.relativeTo(targetDir).invariantSeparatorsPath.replace('/', '-')
+    val targetFile = targetDir.resolve(targetFilename)
+
+    if (!pomFile.renameTo(targetFile)) {
```

Comment:
For safety (esp. on Windows) better use `Files.move(..., ..., StandardCopyOption.ATOMIC_MOVE)`, wrapped in `runCatching {}` to cope with an exception.

Reply:
Done.